notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Maybe add a brief sentence before this cell to explain that 1) the goal will be to predict a "label sequence" of "next words" from a "features sequence" of "previous words", and 2) the "feature" is obtained by shifting the original sequences to the right using the [BOS] token.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>